src/style/types/attribute.rs

```diff
@@ -100,6 +100,17 @@ Attribute! {
     Italic = 3,
     /// Underlines the text.
     Underlined = 4,
+
+    // Other types of underlining
+    /// Double underlines the text.
+    DoubleUnderlined = 2,
```

Comment:
The numbers in here are handled differently than the numbers in the other attributes. These need to be already existing numbers because these attributes print out `\x1b[4:{}m` and the `{}` is a number that's already on the list. The fact that they're already existing numbers shouldn't cause conflicts, from my understanding of the code.

While I could use different numbers and just use the enum index to figure out what `{}` should be, that would probably be less efficient.

Reply:
Thanks for the clarification, yes that makes sense.